src/style.css

```
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
+
+html {
+  font-size: 16px;
+}
+
+.form {
+  margin-top: 20px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0,1);
+  border-radius: 4px;
+  position: relative;
+  background-color: #fff;
+}
+
+.input {
+  font-family: Avenir-Book, sans-serif;
+  width: 100%;
+  height: 100%;
+  border: none;
+  outline: none;
+  font-size: inherit;
+  font-weight: inherit;
+}
+
+input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+input:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  color: #3d4e61;
+  border-radius: 3px;
+}
+
+.form-field {
+  display: block;
+}
+
+.form-field-big {
+  height: 68px;
+}
+
+.form-field-small {
+  height: 40px;
+}
+
+.input-big {
+  padding-left: 62px;
+}
+
+.input-small {
+  font-size: 14px;
+  padding-left: 33px;
+}
+
+.search-but {
```

```suggestion
.search-button {
```
не сокращай слова в названиях классов